docs/UsersGuide/source/WE2Etests.rst

```diff
@@ -3,120 +3,445 @@
 
-.. code-block:: console
+.. _ModExistingTest:
+
+Modifying an Existing Test
+---------------------
+To modify an existing test, simply edit the configuration file for that test by changing
+existing variable values and/or adding new variables to suit the requirements of the
+modified test.  Such a change may also require modifications to the test description
+in the header of the file.
+
+
+.. _AddNewTest:
+
+Adding a New Test
+---------------------
+To add a new test named, for example ``new_test01``, to one of the existing categories listed
+above, say ``wflow_features``:
+
+1) Choose an existing test configuration file in any one of the category directories that
+   matches most closely the new test to be added.  Copy that file to ``config.new_test01.sh``
+   and, if necessary, move it to the ``wflow_features`` category directory. 
+
+2) Edit ``config.new_test01.sh`` so that the header containing the test description properly
+   describes the new test.
+
+3) Further edit ``config.new_test01.sh`` by modifying existing experiment variable values
+   and/or adding new variables such that the test runs with the intended configuration.
+
+
+.. _AddNewCategory:
+
+Adding a New WE2E Test Category
+-----------------------------
+To create a new test category called, e.g. ``new_category``:
+
+1) In the directory ``ufs-srweather-app/regional_workflow/tests/WE2E/test_configs``,
+   create a new directory named ``new_category``. 
+
+2) In the file ``get_WE2Etest_names_subdirs_descs.sh``, add the element ``"new_category"`` 
+   to the array ``category_subdirs`` that contains the list of categories/subdirectories
+   in which to search for test configuration files.  Thus, ``category_subdirs`` becomes:
+
+   .. code-block:: console
+
+     category_subdirs=( \
+       "." \
+       "grids_extrn_mdls_suites_community" \
+       "grids_extrn_mdls_suites_nco" \
+       "wflow_features" \
+       "new_category" \
+       )
+
+New tests can now be added to ``new_category`` using the procedure described in 
+:numref:`Section %s <AddNewTest>`.
+
+
+.. _CreateAltTestNames:
+
+Creating Alternate Names for a Test
+-----------------------------------
+In order to prevent proliferation of WE2E tests, users might want to use the same
+test for multiple purposes.  For example, consider the test 
+
+  ``grid_RRFS_CONUScompact_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16`` 
+
+in the ``grids_extrn_mdls_suites_community`` category.  This checks for the successful
+completion of the Rocoto workflow running the combination 
+of the ``RRFS_CONUScompact_25km`` grid, the ``FV3GFS`` model for
+ICs and LBCs, and the ``FV3_GFS_v16`` physics suite.  If this test also
+happens to use the inline post capability of the weather model (it currently 
+doesn't; this is only a hypothetical example), then this test can also be used
+to ensure that the inline post feature of the App/weather model (which is
+activated in the App by setting ``WRITE_DOPOST`` to ``"TRUE"``) is working properly.
```

Comment:
@gspetro-NOAA - It seems that inline post is just being used as an example here for another purpose (creating new tests), but it does seem to imply that inline post could be a supported feature should a user want to turn it on. I think we need to be clear up front about whether or not it is or isn't supported, instead of having it sporadically appear in different parts of the documentation. The feature is functioning for regional applications now, but earlier this year a decision was made to go with offline only for SRW. Let me discuss with other UPP code managers and get a final decision on this so we don't have to deal with word-smithing documentation like this.

Reply:
@gspetro-NOAA - After discussion with EMC code managers, I think this sort of documentation is ok to keep. The feature works from a technical standpoint, so as long as those in charge of supporting this release are ok with this mention it's fine.

Reply:
@gspetro-NOAA I'll wait to get the ok from you to @fossell's response before merging.  Thanks.

Reply:
@gsketefian You're all set to merge!

Reply:
@gspetro-NOAA Thanks!